pctasks/dataset/pctasks/dataset/items/task.py

```diff
@@ -96,13 +132,16 @@ def _ensure_collection(items: List[pystac.Item]) -> None:
                 ) from e
             if isinstance(result, WaitTaskResult):
                 return result
+            elif result is None:
+                logger.warning(f"No items created from {args.asset_uri}")
```

Comment:
I guess this is a slight change from before when `result` is the empty list. Previously, that would warn about ``No items created from ...`.

Now we'll silently allow that.

I'm trying to decide what behavior we want. In general, I prefer to not treat the empty case specially. Perhaps some dataset has a valid reason for returning `[]` for a specific `create_items`, and we don't want to pollute the logs with noisy warnings.

Reply:
We're actually deliberately returning `[]` in one of the sentinl-3 collections, since we aren't able to filter out a subset of files with just prefix / suffix filtering. We need to open up the asset to determine that we want to skip it.